ddtrace/debugging/_function/discovery.py

```diff
@@ -9,7 +9,7 @@
 try:
     from collections.abc import Iterator
 except ImportError:
-    from collections import Iterator  # type: ignore[attr-defined,no-redef]
+    from collections import Iterator  # type: ignore[attr-defined]
```

Comment:
For some reason this `no-redef` was causing linting failures on this branch. That's why this otherwise-unrelated change is present here.